scripts/release/marketplaceRelease.js

```diff
@@ -0,0 +1,139 @@
+const nodefetch = require("node-fetch");
+const { join } = require("path");
+
+const config = {
+    appStoreUrl: "https://appstore.home.mendix.com/rest/packagesapi/v2"
+};
+
+main().catch(e => {
+    console.error(e);
+    process.exit(-1);
+});
+
+async function main() {
+    const {
+        widgetName,
+        version,
+        marketplace: { minimumMXVersion, marketplaceId }
+    } = require(join(process.cwd(), "package.json"));
+
```

Comment:
Maybe it's nice to add a log for which tag is being released as context, something like
```suggestion
    console.log(`Starting release process for tag ${process.env.TAG}`);

```